thehive4py/endpoints/cortex.py

```diff
@@ -17,7 +24,7 @@ def create_analyzer_job(
 
     def create_responder_action(
         self, object_id: str, object_type: str, responder_id: str
```

Comment:
When checking the corresponding [docs](https://docs.strangebee.com/thehive/api-docs/#tag/Cortex/operation/Create%20an%20action) I can see that it supports more parameters for this endpoint, what about adding an input type hint too? E.g.:

```python
class InputResponderActionRequired(TypedDict):
    responderId: str
    objectType: str
    objectId: str

<...>
```

and then:

```suggestion
        self, action: InputResponderAction
```